### PR TITLE
MWPW-161642: [Loc] Added State management skeleton in new Loc project

### PR DESCRIPTION
**Pull request by Zweihand3r**
* Added project skeleton with root classes and signal store

Resolves: [MWPW-161642](https://jira.corp.adobe.com/browse/MWPW-161642)

**Test URLs:**
- Before: https://main--milo--zweihand3r.hlx.page/drafts/arnab/locui-create?martech=off
- After: https://initial-setup--milo--zweihand3r.hlx.page/drafts/arnab/locui-create?martech=off

### CODE REVIEW

File: libs/blocks/locui-create/utils/state.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  htmlFlow: false,
+  regionalEditBehavior: '',
+  urls: [],
+  fragments: false,
+  locales: [],
+};
+
+export const currentStep = signal(1);
+export const isLoading = signal(false);
+export const project = signal({ ...emptyProject });
```
Comment (nkthakur48) [inline libs/blocks/locui-create/utils/state.js:14]: Could we have a new line character here

File: libs/blocks/locui-create/create/view.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { html } from '../../../deps/htm-preact.js';
+
+export default function Create({}) {
+  return html`<div>Locui Create Project</div>`;
+}
```
Comment (nkthakur48) [inline libs/blocks/locui-create/create/view.js:5]: Could we have a new line character here

File: libs/blocks/locui-create/create/view.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { html } from '../../../deps/htm-preact.js';
+
+export default function Create({}) {
```
Comment (nkthakur48) [inline libs/blocks/locui-create/create/view.js:3]: If `Create` element is not taking any props, then probably we could remove `{}`

**Review by nkthakur48 — CHANGES_REQUESTED** [review]
@Zweihand3r Please have a look at review comments

File: libs/blocks/locui-create/utils/state.js (hunk 1 above)
Comment (Zweihand3r) [reply libs/blocks/locui-create/utils/state.js]: Added new line char

File: libs/blocks/locui-create/create/view.js (hunk 3 above)
Comment (Zweihand3r) [reply libs/blocks/locui-create/create/view.js]: Added new line char

File: libs/blocks/locui-create/create/view.js (hunk 2 above)
Comment (Zweihand3r) [reply libs/blocks/locui-create/create/view.js]: Added new line char

**Review by nkthakur48 — APPROVED** [review]

**Review by maagrawal16 — APPROVED** [review]

**Review by hnvadobe — APPROVED** [review]

**Pull request merged**